### ISSUE TITLE
DAX script automatic comments are saved back as comments in DAX expressions

### ISSUE THREAD

**Issue by bernatagulloesbrina** [TabularEditor/TabularEditor3#331]
I DAX-scripted a calculation group table that also contains some measures. Then the script starts with the measures, and then goes to the calculation group, but before that a heading is added for clarity (which is great) 

```
--------------------------------
-- Measure: [CF Dynamic Measure]
--------------------------------
MEASURE 'Dynamic Measure'[CF Dynamic Measure] = BLANK()

---------------------------------------
-- Calculation Group: 'Dynamic Measure'
---------------------------------------
CALCULATIONGROUP 'Dynamic Measure'[Measure]

    CALCULATIONITEM "Margin" = 
        IF(
            ISSELECTEDMEASURE( [Dynamic Measure],[CF Dynamic Measure] ),
            [Margin],
            SELECTEDMEASURE( )
        )
        FormatString = ...
```

The thing comes when you synch and commit. 
Back in PBID you see this: 
![image](https://user-images.githubusercontent.com/40455069/147373250-f4efd950-3f2c-4444-b7a2-639f02897616.png)

I guess the automatic comments should start with some special sequence of chars that can be recognized so that such comments are not saved in the measures preceding them 

Thank you for all your efforts and looking forward the dax debugger! 

Regards!

**Comment by bernatagulloesbrina** [comment]
also not clear why the "BLANK()" part of the measure comes on the second row as this is not really a multi-line expression 

![image](https://user-images.githubusercontent.com/40455069/147373289-e6b9ee52-9bb6-41fa-b652-238bffa1ca8a.png)

**Comment by bernatagulloesbrina** [comment]
oh, well I guess it is the moment it picks up the comment as part of the expression !

**Comment by otykier** [comment]
Fixed in [3.2.1](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.2.1). Thanks for reporting!
